### ISSUE TITLE
Error when trying to send doge

### ISSUE THREAD

**Issue by smilledge** [langerhans/multidoge#42]
I'm getting the following error when attempting to send any amount of doge;

```
The send of your dogecoin failed. The error was "java.lang.IllegalArgumentException Cannot convert using a negative conversion multiplier".
```

I assume this has something to do with all values from Cryptsy returning negatives?

![suchbug](https://cloud.githubusercontent.com/assets/1536407/11452315/30ecf2de-962f-11e5-8010-cb7b5ffb640a.png)

**Comment by langerhans** [comment]
Yeah, pretty sure that is the issue. Cryptsy might have changed their API so the old old Version of XChange we used here doesn't deal with it correctly anymore.  
A workaround can be to go to Preferences, and disabled "Show dogecoin converted to fiat currency". Can you maybe try that?

**Comment by smilledge** [comment]
Yeah that fixed it, thanks!

**Comment by Hectate** [comment]
I didn't even think to check Github to file an issue until just replying in the reddit thread. This works for me as well, thanks.

**Comment by langerhans** [comment]
This is fixed in 0.1.7 as I removed Cryptsy completely now.
